src/js/components/DescriptionList.js

```diff
@@ -1,6 +1,12 @@
+import PureRenderMixin from 'react-addons-pure-render-mixin';
 import React from 'react';
 
 class DescriptionList extends React.Component {
+  constructor() {
+    super(...arguments);
+    this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate;
```

Comment:
Shouldn't the `PureRenderMixin.shouldComponentUpdate` be bound?
this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate.bind(this);